jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/DockerContextTaskTest.java

```diff
@@ -120,6 +122,16 @@ public void testGenerateDockerContext_errorOnWindowsAppRootWithDriveLetter() {
     }
   }
 
+  @Test
+  public void testGenerateDockerContext_env() throws IOException {
+    task.generateDockerContext();
+
+    Path dockerfile = projectRoot.getRoot().toPath().resolve("build/jib-docker-context/Dockerfile");
```

Comment:
I think `getDockerfileLine("ENV")` will work, as long as you care to get the first ENV line only.

Reply:
I think that's only available in the integration tests?

Reply:
I see the method at the end of this file? https://github.com/GoogleContainerTools/jib/blob/master/jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/DockerContextTaskTest.java#L160